src/test_pages/TimelinePage.ts

```diff
@@ -34,12 +40,49 @@ export default class TimelinePage extends BasePage {
   }
 
   public async closeLineNumber() {
-    await this.clickOnElement(this.close_line_number)
+    await this.clickOnElement(this.close_line_number, 3000)
   }
 
   public async verifyRouteSelectionVisible(isVisible: boolean) {
     isVisible
-      ? await expect(this.route_select).toBeVisible({ timeout: 2000 })
+      ? await expect(this.route_select).toBeVisible({ timeout: 3000 })
       : await expect(this.route_select).toBeHidden({ timeout: 2000 })
```

Comment:
I have a good way to shorten the code, I need to know if it is ok to use getter for locators,
and this was before i worked because of that i not changed it and this is my mistake

Reply:
and if i create getter 
```
public get operatorsDropdown() {
    return this.operators_dropdown
}
```
this ok?

Reply:
I like this approach.

Reply:
you can use the getter, and remove the field

Reply:
I would do it this way: 

public async verifyRouteSelectionVisible(isVisible: Boolean, timeout: 2000) {
	await this.verifySelectionVisible(this.route_select, isVisible, timeout)
}

But finally Noam decides if it is acceptable or not.

Reply:
@DanLeshinsky I think this PR can be merged,
However, I did left some comments, all of them are just personal opinions.
please see this one - I think you'll agree with me about it
https://github.com/hasadna/open-bus-map-search/pull/416#discussion_r1465566666